wagtail/admin/tests/test_workflows.py

```diff
@@ -190,3 +190,128 @@ def test_post(self):
         self.assertEqual(WorkflowPage.objects.filter(workflow=self.workflow, page=self.other_page).count(), 1)
 
 
+class TestRemoveWorkflow(TestCase, WagtailTestUtils):
+    fixtures = ['test.json']
+
+    def setUp(self):
+        self.login()
+        self.workflow = Workflow.objects.create(name="workflow")
+        self.page = Page.objects.first()
+        WorkflowPage.objects.create(workflow=self.workflow, page=self.page)
+
+    def post(self, post_data={}):
+        return self.client.post(reverse('wagtailadmin_workflows:remove', args=[self.workflow.id, self.page.id]), post_data)
+
+    def test_post(self):
+        # Check that a WorkflowPage instance is removed correctly
+        response = self.post()
+        self.assertEqual(WorkflowPage.objects.filter(workflow=self.workflow, page=self.page).count(), 0)
+
+
+class TestTaskIndexView(TestCase, WagtailTestUtils):
+
+    def setUp(self):
+        self.login()
+
+    def get(self, params={}):
+        return self.client.get(reverse('wagtailadmin_workflows:task_index'), params)
+
+    def test_simple(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'wagtailadmin/workflows/task_index.html')
+
+        # Initially there should be no tasks listed
+        self.assertContains(response, "No tasks have been created.")
+
+        SimpleTask.objects.create(name="test_task", active=True)
+
+        # Now the listing should contain our task
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'wagtailadmin/workflows/task_index.html')
+        self.assertNotContains(response, "No tasks have been created.")
+        self.assertContains(response, "test_task")
+
+    def test_deactivated(self):
+        Task.objects.create(name="test_task", active=False)
+
+        # The listing should contain our workflow, as well as marking it as disabled
```

Comment:
workflow -> task. Also, as above, we should test the full HTML tag rather than just the string "disabled"

Reply:
Thanks, I've changed it over now (and added the tag check).